### PR TITLE
Add a CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jibsheet**
https://help.github.com/articles/adding-support-resources-to-your-project/

I took Ned's boilerplate

@nedbat @gsong Any objections / thoughts about this?

### CODE REVIEW

**Comment by jibsheet** [comment]
@edx/devops please review

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1 @@
+GitHub issues are not the best place to get support with OpenEDX. The mailing list and Slack channels are explained here: http://open.edx.org/getting-help. If it turns out there's a bug in the configuration scripts, we can open an issue or PR here.
```
Comment (nedbat) [inline CONTRIBUTING.md]: Let's spell it properly: "Open edX".

Also, this isn't about Contributing exactly.... though I think it is a good message to get in front of people.

**Review by nedbat — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (gsong) [inline CONTRIBUTING.md]: I would word it even more strongly:

> **Do not** use GitHub issues for Open edX support. Instead, use the mailing list or Slack channels as explained in [Getting Help](https://open.edx.org/getting-help#where_to_ask_questions). If you do find a bug, feel free to open an issue or PR in this repo.

**Comment by jibsheet** [comment]
Agreed - not 100% contributing, but I saw their blog and thought "great, maybe I can redirect even more people".  Thanks for the edit requests.

File: CONTRIBUTING.md (hunk 1 above)
Comment (nedbat) [reply CONTRIBUTING.md]: It's still not spelled right... :(

**Comment by jibsheet** [comment]
@nedbat hopefully better now

**Comment by nedbat** [comment]
"Open edX"

**Comment by jibsheet** [comment]
@nedbat Ope NedX, right?  I think it's right this time.

**Review by nedbat — APPROVED** [review]

**Pull request merged**